src/styles/components.css

```diff
@@ -143,7 +134,7 @@
   }
 
   .check-field {
-    @apply grid grid-cols-[auto_1fr] gap-x-3 items-center;
+    @apply grid grid-cols-[auto_1fr] justify-items-start gap-x-3 items-center;
```

Comment:
this reduces click area of the label to the text itself and not the whole line - as its confusing that clicking an empty space triggers the checkbox
before:
<img width="999" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/0415ff51-080b-446a-a91b-c29aaba957d6">

now: 
<img width="386" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/e71a61c0-d8b6-4b97-90ff-81f7ea369374">